### PR TITLE
fix balance thread hang up

### PR DESCRIPTION
**Pull request by critical27**
If future's callback is invoked before the future is set. And in the callback, baton will wait forever.

### CODE REVIEW

File: src/meta/processors/admin/AdminClient.cpp [hunk 1]
```diff
@@ -207,23 +207,21 @@ folly::Future<Status> AdminClient::updateMeta(GraphSpaceID spaceId,
...
-                                         apache::thrift::util::enumNameSafe(code).c_str()));
-            }
+    kv_->asyncMultiPut(kDefaultSpaceId, kDefaultPartId, std::move(data),
+        [this, p = std::move(pro)] (nebula::cpp2::ErrorCode code) mutable {
+            folly::via(ioThreadPool_.get(), [code, p = std::move(p)]() mutable {
+                if (code == nebula::cpp2::ErrorCode::SUCCEEDED) {
+                    p.setValue(Status::OK());
+                } else {
+                    p.setValue(Status::Error("Access kv failed, code: %s",
+                                            apache::thrift::util::enumNameSafe(code).c_str()));
```
Comment (darionyaphet) [inline src/meta/processors/admin/AdminClient.cpp:222]: sorry to say alignment.

Comment (critical27) [reply src/meta/processors/admin/AdminClient.cpp:222]: 😂 my poor vscode, hold on

**Review by darionyaphet — APPROVED** [review]

File: src/meta/processors/admin/HBProcessor.cpp [hunk 2]
```diff
@@ -43,8 +43,8 @@ void HBProcessor::process(const cpp2::HBReq& req) {
        }
     }
 
-    VLOG(3) << "Receive heartbeat from " << host
-            << ", role = " << apache::thrift::util::enumNameSafe(req.get_role());
+    LOG(INFO) << "Receive heartbeat from " << host
```
Comment (bright-starry-sky) [inline src/meta/processors/admin/HBProcessor.cpp:46]: VLOG（1）？

File: src/kvstore/wal/FileBasedWal.cpp [hunk 3]
```diff
@@ -315,7 +315,7 @@ void FileBasedWal::closeCurrFile() {
     timebuf.modtime = currInfo_->mtime();
     timebuf.actime = currInfo_->mtime();
     VLOG(1) << "Close cur file " << currInfo_->path() << ", mtime: " << currInfo_->mtime();
-    CHECK_EQ(utime(currInfo_->path(), &timebuf), 0);
```
Comment (bright-starry-sky) [inline src/kvstore/wal/FileBasedWal.cpp:318]: what's reason? no need to check the system error?

**Review by bright-starry-sky — COMMENTED** [review]
great job，It's a difficult problem to identify.

File: src/meta/processors/admin/HBProcessor.cpp (hunk 2 above)
Comment (critical27) [reply src/meta/processors/admin/HBProcessor.cpp:46]: Previously was INFO, modified to VLOG(3) by others. It is the most convenient way to tell which is meta leader.

**Review by liuyu85cn — APPROVED** [review]

**Pull request merged**